pkg/asset/installconfig/azure/dns.go

```diff
@@ -136,8 +166,43 @@ func (client *ZonesClient) GetAllPublicZones() (map[string]string, error) {
 		//the property is present in the REST api response, but not mapped yet in the stable SDK (present in preview)
 		//https://github.com/Azure/azure-sdk-for-go/blob/07f918ba2d513bbc5b75bc4caac845e10f27449e/services/preview/dns/mgmt/2018-03-01-preview/dns/models.go#L857
 		for _, zone := range zonesPage.Values() {
-			allZones[to.String(zone.Name)] = to.String(zone.ID)
+			zoneName := to.String(zone.Name)
+			allZones[zoneName] = append(allZones[zoneName], to.String(zone.ID))
 		}
 	}
 	return allZones, nil
 }
+
+/*
+
+func IsForbidden(err error) bool {
+	requestError, ok := err.(awserr.RequestFailure)
+	return ok && requestError.StatusCode() == http.StatusForbidden
+}
+
+// IsErrorThrottle returns whether the error is to be throttled based on its
+// code.
+// Returns false if error is nil.
+func IsErrorThrottle(err error) bool {
+	if aerr, ok := err.(awserr.Error); ok && aerr != nil {
+		return isCodeThrottle(aerr.Code())
+	}
+	return false
+}
+
+
+// IsForbidden checks whether a response from the GPC API was forbidden,
+// indicating that a given service account cannot access the specified project.
+func IsForbidden(err error) bool {
+        gErr, ok := err.(*googleapi.Error)
+        return ok && gErr.Code == 403
+}
+
+// IsThrottled checks whether a response from the GPC API returns Too Many
+// Requests
+func IsThrottled(err error) bool {
+        gErr, ok := err.(*googleapi.Error)
+        return ok && gErr.Code == 429
+}
+
+*/
```

Comment:
can you drop all this commented code

Reply:
These were left in by accident. I have removed them.